website/docs/r/third_party_integration.markdown

```diff
@@ -57,7 +57,7 @@ Additional values based on Type
   * `service_key` - Your Service Key.
 * `DATADOG`
   * `api_key` - Your API Key.
-  * `region` - Indicates which API URL to use, either US or EU. Datadog will use US by default.    
+  * `region` - Indicates which API URL to use, either US, EU, US3, or US5. Datadog will use US by default.    

```

Comment:
@evertsd can you confirm that code changes to this TF resource or to Atlas Go-Client is required to support additional regions?

Reply:
Yep, no code changes required to the tf resource or atlas go-client. I was able to execute an example with the US5 region and have everything go through successfully.

Reply:
nice!